Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/OnOffPeriod.mo

```diff
@@ -0,0 +1,131 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.Relay;
+block OnOffPeriod
+  "Calculate the lengths of the on period and the off period"
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput tim(
+    final quantity="Time",
+    final unit="s")
+    "Simulation time"
+    annotation (Placement(transformation(extent={{-140,20},{-100,60}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput On
```

Comment:
Use lower case as that is the convention. Being consistent helps filtering results.